### PR TITLE
Ext: Update to SVT-AV1 v1.7.0, zlibpng to 1.3 and libpng to 1.6.40

### PR DESCRIPTION
**Pull request by vrabaud**

### CODE REVIEW

**Review by wantehchang — APPROVED** [review]
LGTM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+* Update svt.cmd/svt.sh: v1.7.0
+* Update zlibpng.cmd: zlib 1.3 and libpng 1.6.40
+
```
Comment (wantehchang) [inline CHANGELOG.md:12]: When you resolve the merge conflicts, please make sure this goes before the new 1.0.1 section.

Comment (vrabaud) [reply CHANGELOG.md:12]: Of course, done.

**Review by wantehchang — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**